### PR TITLE
[Consumer WARP] Remove WARP+ mode

### PR DESCRIPTION
**Pull request by ranbel**
PCX-14248

### CODE REVIEW

**Review by patriciasantaana — APPROVED** [review]

File: src/content/docs/warp-client/warp-modes.mdx [hunk 1]
```diff
@@ -41,22 +41,8 @@ This will enable the **WARP via Local Proxy** option in the **WARP Settings** me
...
 
-## WARP+
+## WARP+ Unlimited
 
-While WARP can take advantage of the many Cloudflare data centers around the world to give you a more private and robust connection, WARP+ subscribers get access to a larger network. More cities to connect to means you are likely to be closer to a Cloudflare data center – which can reduce the latency between your device and Cloudflare, and improve your browsing speed. As a result, sites load faster, both for those on the Cloudflare network and those that are not.
+WARP+ Unlimited is our monthly subscription offering that provides access to a larger network of Cloudflare data centers. More cities to connect to means you are likely to be closer to a Cloudflare data center. This can reduce the latency between your device and Cloudflare and improve your browsing speed. As a result, sites load faster, both for those on the Cloudflare network and those that are not.
```
Comment (kokolocomotion1) [inline src/content/docs/warp-client/warp-modes.mdx]: @ranbel can you reword it to this please?

While WARP can take advantage of the many Cloudflare data centers around the world to give you a more private and robust connection, WARP+ Unlimited subscribers get access to a larger network. More cities to connect to means you are likely to be closer to a Cloudflare data center – which can reduce latency and improve your browsing speed.

WARP+ Unlimited is a paid, monthly subscription that can be purchased via the Apple App Store and Google Play Store.

**Review by kokolocomotion1 — APPROVED** [review]
